scripts/release

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env node
+
+const execSync = require('child_process').execSync;
+const minimist = require('minimist');
+const standardVersion = require('standard-version');
+const chalk = require('chalk');
+const args = minimist(process.argv.slice(2));
+
+const DEFAULT_UPSTREAM_REPO_ORIGIN = 'origin';
+const DEFAULT_UPSTREAM_REPO_BRANCH = 'master';
+const DEFAULT_PRERELEASE_DIST_TAG = 'next';
+const origin = args['repo-origin'] || DEFAULT_UPSTREAM_REPO_ORIGIN;
+const branch = args['repo-branch'] || DEFAULT_UPSTREAM_REPO_BRANCH;
+const prereleaseTag = args['prerelease-tag'] || DEFAULT_PRERELEASE_DIST_TAG;
```

Comment:
@michaelsbradleyjr dist tag is now configurable using `--prerelease-tag`. Shall we rename this to `--npm-dist-tag`?